operations/scale-to-one-az-addon-parallel-cats.yml

```diff
@@ -0,0 +1,13 @@
+---
+# Use this after the scale-to-one-az ops file to scale up to minimal size
+# in order to run CATS quickly without flakes and 12 threads
+- type: replace
```

Comment:
@dsboulder what do you think about placing the CATs ops file in the `test` directory with the other ops files that are for testing, rather than production, purposes?

Reply:
Yep, great idea. Done!